### PR TITLE
style: update descriptions style

### PR DESCRIPTION
**Pull request by kooriookami**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/descriptions/src/descriptions-cell.ts [hunk 1]
```diff
@@ -55,7 +55,10 @@ export default defineComponent({
             class: [
               'el-descriptions__cell',
               'el-descriptions__label',
-              { 'is-bordered-label': this.descriptions.border },
+              {
+                'is-bordered-label': this.descriptions.border,
+                'is-vertical-label': this.descriptions.direction === 'vertical',
+              },
```
Comment (JeremyWuuuuu) [inline packages/components/descriptions/src/descriptions-cell.ts:61]: Suggestion:

```
const { border, direction } = this.description
```

File: packages/components/descriptions/src/index.vue [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <template>
-  <div class="el-descriptions">
+  <div
+    class="el-descriptions"
+    :class="descriptionsSize ? `el-descriptions--${descriptionsSize}` : ''"
```
Comment (JeremyWuuuuu) [inline packages/components/descriptions/src/index.vue:4]: **Suggestion:** Can we merge them into a computed?

```ts
const prefix = 'el-descriptions'
const descriptionKls = computed(() => [prefix, descriptionSize.value ? `${prefix}--${descriptionSize.value}` : '']
```

**Review by jw-foss — COMMENTED** [review]
Left 2 comments for you to consider, thanks.

File: packages/components/descriptions/src/descriptions-cell.ts [hunk 3]
```diff
@@ -56,13 +57,13 @@ export default defineComponent({
...
-                'is-bordered-label': this.descriptions.border,
-                'is-vertical-label': this.descriptions.direction === 'vertical',
+                'is-bordered-label': border,
+                'is-vertical-label': direction === 'vertical',
               },
               labelAlign,
               labelClassName,
             ],
-            colSpan: this.descriptions.direction === 'vertical' ? span : 1,
+            colSpan: direction === 'vertical' ? span : 1,
```
Comment (JeremyWuuuuu) [inline packages/components/descriptions/src/descriptions-cell.ts:66]: One more thing: we have multiple `direction === 'vertical'` here in this file, we should make it variable:

```ts
const isVertical = direction === 'vertical' 
```

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
LGTM now, thanks!

**Pull request merged**